lib/transforms/json-to-html-data.js

```diff
@@ -284,6 +308,30 @@ function getNotes (data) {
   }
 }
 
+function getObjectType (data) {
+  var value;
+  if (data.attributes.name) {
+    data.attributes.name.forEach(el => {
+      if (el.primary) {
+        value = el.value || null;
+      }
+    });
+  }
+  return value ? {key: 'type', value: value} : null;
+}
+
+function getTaxonomy (resource) {
+  const taxonomy = [];
```

Comment:
same here is this really a `const` ?
https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Statements/const